WindowsFormsApp2/AddCostForm.cs

```diff
@@ -24,24 +21,22 @@ private void CancelAddButton_Click(object sender, EventArgs e)
 
         public void AddCostButton_Click(object sender, EventArgs e)
         {
-
-            //cost.name = newCostNameInput.Text;
-            cost.cost = Convert.ToDecimal(PriceInput.Value);
-            cost.comment = commentInput.Text;
+            Expense cost = new Expense();
+            cost.Cost = Convert.ToDecimal(PriceInput.Value);
+            cost.Comment = commentInput.Text;
             if (Convert.ToString(newCostCategory.SelectedItem) == "Traffic")
             { cost.Category = "Traffic"; }
             if (Convert.ToString(newCostCategory.SelectedItem) == "Utilities")
             { cost.Category = "Utilities"; }
             if (Convert.ToString(newCostCategory.SelectedItem) == "Services Communication")
-            { cost.Category = "Services Communication"; }
```

Comment:
Please, check this:
https://stackoverflow.com/questions/14248303/c-sharp-how-to-show-name-and-insert-values-names-id-from-combobox-in-c-sharp

Fix it in a separate pull request.